tests/Backend/SOX/ExportTableTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\SOX;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\TableExporter;
+use Keboola\Test\StorageApiTestCase;
+
+class ExportTableTest extends StorageApiTestCase
+{
+    private Client $developerClient;
+
+    private string $downloadPath;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->developerClient = $this->getDeveloperStorageApiClient();
+        $this->branches = new DevBranches($this->developerClient);
+        foreach ($this->getBranchesForCurrentTestCase() as $branch) {
+            $this->branches->deleteBranch($branch['id']);
+        }
+        $this->downloadPath = $this->getExportFilePathForTest('languages.sliced.csv');
+    }
+
+    /**
+     * @dataProvider tableExportData
+     */
+    public function testTableAsyncExportInDevBranch(CsvFile $importFile, string $expectationsFileName, array $exportOptions = []): void
```

Comment:
imho by tady v tom testu mohlo byt par komentaru, ktere rikaji, co se tam snazis delat. Jinak se musi nahodit v hlavne interpreter a to boli

Reply:
Pridal som tam len komentare k exportom co prejdu aby som popisal o co sa tam snazim, ostatne miesta sa snazim popisovat hlavne v `$this->fail(...)` a vysvetlit preco by to malo spadnut tak mi pridu komentare na viac :)